aim/storage/sdk/repo.py

```diff
@@ -35,14 +35,15 @@ def __init__(self, path: str, *, read_only: bool = None):
         self.container_view_pool: Dict[ContainerConfig, ContainerView] = WeakValueDictionary()
 
         os.makedirs(self.path, exist_ok=True)
-        # os.makedirs(os.path.join(self.path, 'chunks'), exist_ok=True)
-        # os.makedirs(os.path.join(self.path, 'locks'), exist_ok=True)
-        # os.makedirs(os.path.join(self.path, 'progress'), exist_ok=True)
 
-        self.meta_tree = self.request('meta', read_only=True, from_union=True).tree().view('meta')
+        self._meta_tree = None
```

Comment:
is `_meta_tree` used?

Reply:
`meta_tree` is property right now, to make it lazier.

Reply:
I mean _meta_tree. the one with _. I cannot find any usage for it.